src/sql-parser/src/parser.rs

```diff
@@ -7053,6 +7053,9 @@ impl<'a> Parser<'a> {
         if self.parse_keyword(TIMESTAMP) {
             self.parse_explain_timestamp()
                 .map_parser_err(StatementKind::ExplainTimestamp)
+        } else if self.parse_keyword(KEY) || self.parse_keyword(VALUE) {
```

Comment:
Here you could use 
```Rust
else if self.peek_keyword(KEY) || self.peek_keyword(VALUE)
```
to avoid the call to `self.prev_token();` later.

Alternatively, you could do something like
```Rust
else if let Some(schema_for) = self.parse_one_of_keywords(&[KEY, VALUE]) {
  self.parse_explain_schema(schema_for).map_parser_err(StatementKind::ExplainSchema)
}
```

Reply:
done!